[tado] Add support for fanLevel, verticalSwing, horizontalSwing, light API (#12470)
openhab/openhab-addons: bundles/org.openhab.binding.tado/README.md

`operationMode` | String | Operation mode the zone is currently in. One of `SCHEDULE` (follow smart schedule), `MANUAL` (override until ended manually), `TIMER` (override for a given time), `UNTIL_CHANGE` (active until next smart schedule block or until AWAY mode becomes active) | RW | `HEATING`, `AC`, `DHW`
`batteryLowAlarm` | Switch | A control device in the Zone has a low battery (if applicable) | R | Any Zone
`openWindowDetected` | Switch | An open window has been detected in the Zone | R | Any Zone
`light` | Switch | State (`ON`, `OFF`) of the control panel light (if applicable) | RW | `AC`

The `RW` items are used to either override the schedule or to return to it (if `hvacMode` is set to `SCHEDULE`).

<sup>1)</sup> Simpler A/C units have fan speed settings in the range [`LOW`, `MIDDLE`, `HIGH`].
However more sophisticated devices have settings in the range [`SILENT`, `LEVEL1`, `LEVEL2`, `LEVEL3`, `LEVEL4`].
So you need to choose the respective Channel type name that matches the features of your device.

<sup>2)</sup> Simpler A/C units have a single switch type swing function that is either `ON` or `OFF`.

> However more sophisticated devices may have either a vertical swing, a horizontal swing, or both, which could also have more complex settings.

Reviewer: ```suggestion
However, more sophisticated devices may have either a vertical swing, a horizontal swing, or both, which could also have more complex settings.
```

Author: Ok.

For example the horizontal swing function could simply be `ON` or it could have more complex settings in the range [`LEFT`, `MID_LEFT`, `MID`, `MID_RIGHT`, `RIGHT`].

> So you need to choose the respective Channel type name that matches the features of your device.

Reviewer: ```suggestion
So you need to choose the respective channel type name that matches the features of your device.
```

Author: Ok.

<sup>3)</sup> The _'one of'_ list contains all possible state values supported within the tado° binding.

> However in reality different A/C units might only support a **_sub-set_** of those values.

Reviewer: ```suggestion
However, in reality different A/C units might only support a **_sub-set_** of those values.
```

Author: Ok.

> And indeed the sub-set of supported values might depend on the current state of the `acPower` and `hvacMode` Channels.

Reviewer: ```suggestion
And indeed the subset of supported values might depend on the current state of the `acPower` and `hvacMode` channels.
```

Author: Ok.

> In that case, if you send a Channel command value to an A/C unit which does not (currently) support that particular state value, then openHAB will report a '422' run-time error in the log.

Reviewer: ```suggestion
In that case, if you send a channel command value to an A/C unit which does not (currently) support that particular state value, then openHAB will report a '422' run-time error in the log.
```

Author: Ok.
